include/seqan/index/index_fm_rank_dictionary_levels.h

```diff
@@ -505,7 +505,8 @@ struct RankDictionary<TValue, Levels<TSpec, TConfig> >
         // TODO(cpockrandt): make all bitmasks a const-expr
         const unsigned maxValue = (1 << _BITS_PER_VALUE) - 1;
         const unsigned padding = _BITS_PER_WORD % _BITS_PER_VALUE;
```

Comment:
don't use the max metafunction, use the stl's max function instead and use it up here. You could also make it constexpr